pygmt/tests/test_blockm.py

```diff
@@ -96,3 +96,14 @@ def test_blockmean_without_outfile_setting():
     assert isinstance(output, pd.DataFrame)
     assert output.shape == (5849, 3)
     npt.assert_allclose(output.iloc[0], [245.888877, 29.978707, -384.0])
+
+
+def test_blockmode_input_dataframe(dataframe):
+    """
+    Run blockmode by passing in a pandas.DataFrame as input.
+    """
+    output = blockmode(table=dataframe, spacing="5m", region=[245, 255, 20, 30])
+    assert isinstance(output, pd.DataFrame)
+    assert all(dataframe.columns == output.columns)
+    assert output.shape == (5849, 3)
+    npt.assert_allclose(output.iloc[0], [245.888877, 29.978707, -385.0])
```

Comment:
```suggestion
    npt.assert_allclose(output.iloc[0], [245.88819, 29.97895, -385.0])
```

I forget that the x, y results would also be different for blockmode relative to blockmean.